src/components/course-calendar/month.cjsx

```diff
@@ -29,7 +30,7 @@ CourseMonth = React.createClass
     router: React.PropTypes.func
 
   getInitialState: ->
-    date: @props.startDate or moment()
+    date: @props.startDate or moment(TimeStore.getNow())
```

Comment:
Are there other places where `moment()` should be converted to `moment(TimeStore.getNow())`?